### PR TITLE
Help text for all pages apart from contact

### PR DESCRIPTION
**Pull request by tejpowar**
# Description

Add help and support to all the pages apart from contact https://infinityworks.atlassian.net/browse/TFN-624

# Testing instructions

Navigate through a journey and check to see if the help and support section is there.

It should appear on the contact us page

# Type of change

-   [x] feat - A new feature
-   [ ] fix - A bug fix
-   [ ] docs - Documentation only changes
-   [ ] codestyle - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] refactor - A code change that neither fixes a bug nor adds a feature
-   [ ] perf - A code change that improves performance
-   [ ] test - Adding missing tests or correcting existing tests
-   [ ] build - Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
-   [ ] ci - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] chore - Other changes that don't modify src or test files
-   [ ] revert - Reverts a previous commit
-   [ ] content - Changes to content or copy

# Checklist:

-   [x] Able to run pr locally
-   [n/a] Lighthouse performed (Accessibility 90+ minimum)
-   [x] Followed acceptance criteria
-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have made corresponding changes to the documentation if applicable
-   [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: src/components/Help.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React, { ReactElement } from 'react';
+
+const Help = (): ReactElement => {
+    return (
+        <>
+            <legend className="govuk-fieldset__legend govuk-fieldset__legend--s">
+                <h1 className="govuk-fieldset__heading">Help and Support</h1>
+            </legend>
+            <p className="govuk-body">
+                If you are having problems, please contact the Fares Data Build service through the link below:
```
Comment (dannydaviesIW) [inline src/components/Help.tsx:10]: Can we change the word 'through' in this text to be either 'via' or 'using'?

File: src/components/Help.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+                <div>
+                    <a href="/contact" className="govuk-link">
+                        Contact the FDBT team
```
Comment (dannydaviesIW) [inline src/components/Help.tsx:13]: Also, should this now be FDB (Fares Data Build) team? Or maybe FDBS (Fares Data Build Service) team?

File: tests/components/Help.test.tsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+import Help from '../../src/components/Help';
+
+describe('Help', () => {
+    it('should render the Help component', () => {
+        const wrapper = shallow(<Help />);
+        expect(wrapper).toMatchSnapshot();
+    });
+});
```
Comment (dannydaviesIW) [inline tests/components/Help.test.tsx:10]: Should we have a test to check that the Help component isn't rendered when the contact page is being shown?

Comment (tejpowar) [reply tests/components/Help.test.tsx:10]: yeah can add this

File: src/components/Help.tsx (hunk 2 above)
Comment (tejpowar) [reply src/components/Help.tsx:13]: Have discussed and amended in ticket

**Review by dannydaviesIW — APPROVED** [review]

**Pull request merged**